### PR TITLE
chore: use new amountMath and fix linting errors.

### PR DESCRIPTION
**Pull request by katelynsills**
chore: use new amountMath and fix linting errors.

Turn off linting for UI until we replace with UI from dapp-treasury. Too much to fix for code that will be deleted.

### CODE REVIEW

File: ui/src/components/Wallet.jsx [hunk 1]
```diff
@@ -33,17 +33,31 @@ export default function Wallet() {
...
+              amount: { value },
+            }) => (
+              <ListItem key={pursePetname} value={pursePetname} divider>
+                <ListItemIcon className={classes.icon}>
+                  <PurseIcon />
+                </ListItemIcon>
+                <ListItemText
+                  primary={pursePetname}
+                  secondary={
+                    <>
```
Comment (Chris-Hibbert) [inline ui/src/components/Wallet.jsx:50]: Not a blocker, just my curiousity. What does an empty element do in jsx?

File: package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
   "workspaces": [
     "api",
     "contract",
+    "ui",
```
Comment (Chris-Hibbert) [inline package.json:10]: can we exclude `/src/components/` so eslint doesn't have to look at jsx files?  Or is it an improvement to have those covered by eslint?

File: ui/src/utils/defaults.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
-  "BRIDGE_URL": "http://127.0.0.1:8000",
-  "API_URL": "http://127.0.0.1:8000",
-  "CONTRACT_NAME": "simple-exchange",
-  "INSTALLATION_BOARD_ID": "14561541"
+// GENERATED FROM /Users/katesills/code/dapp-simple-exchange/api/deploy.js
+export default {
+  INSTANCE_BOARD_ID: '605437304',
+  INVITATION_BRAND_BOARD_ID: '1667979430',
+  ASSET_BRAND_BOARD_ID: '262188032',
+  PRICE_BRAND_BOARD_ID: '1202180815',
```
Comment (Chris-Hibbert) [inline ui/src/utils/defaults.js:6]: Should this generated file be included?

**Review by Chris-Hibbert — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (katelynsills) [reply package.json:10]: Eslint has settings for JSX, so it's expected to lint those.

File: ui/src/utils/defaults.js (hunk 3 above)
Comment (katelynsills) [reply ui/src/utils/defaults.js:6]: Hmm, good point. I think not.

File: ui/src/components/Wallet.jsx (hunk 1 above)
Comment (katelynsills) [reply ui/src/components/Wallet.jsx:50]: It's creating a JSX element. In this case it looks like we wanted to have a parent element for the siblings `<b>` and `<i>`

File: ui/src/utils/defaults.js (hunk 3 above)
Comment (katelynsills) [reply ui/src/utils/defaults.js:6]: Ah, I didn't include it in this PR. I think the issue is known generally, but I'll make an issue about .gitignoring `defaults.js` while still ensuring the directory exists for everyone

Comment (katelynsills) [reply ui/src/utils/defaults.js:6]: https://github.com/Agoric/dapp-simple-exchange/issues/29

**Pull request merged**